apps/web/controllers/contributors/show.rb

```diff
@@ -3,11 +3,14 @@ class Show
     include Web::Action
     include Hanami::Pagination::Action
 
-    expose :contributor, :commits
+    expose :contributor, :commits, :commits_count
 
     def call(params)
       @contributor = ContributorRepository.new.find_by_github(params[:id])
-      @commits = all_for_page(CommitRepository.new.all_for_contributor(@contributor.id))
+      commits = CommitRepository.new.all_for_contributor(@contributor.id)
+
+      @commits_count = commits.to_a.size
```

Comment:
you can use faster way to get count :)
```
=> #<CommitRepository relations=[:commits]>
>> repo.root.count
[contributors] [INFO] [2018-02-07 12:07:03 +0300] (0.032178s) SELECT count(*) AS "count" FROM "commits" LIMIT 1
=> 6094
```

Reply:
just call `commits.count`, because `commits` is relation as `repo.root`

Reply:
Nice, I didn't know if I could use `count` here. I should have tried. Thank you.